qa/format.sh

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+# Utility file to run pre-commit hooks locally
+# Usage: bash qa/format.sh
+
+set -e
+
```

Comment:
It would be convenient to make this independent of the current working dir:
```suggestion

: "${TE_PATH:=/opt/transformerengine}"

cd $TE_PATH
```

Reply:
I incorporated this but changed the default path to current dir since we don't run this script anywhere in CI directly. It's just a nifty tool mostly to be used locally since some folks were asking about this.